### DISCUSSION TITLE
最近老是有这种报错，是什么问题？

### DISCUSSION THREAD

**Discussion by winsphinx** [XTLS/Xray-core#3775] [General]
> xray  | 2024/09/08 14:04:32 [Warning] [549252368] app/proxyman/inbound: connection ends > proxy/http: failed to read http request > malformed HTTP request "\x00"

这是什么引起的？

**Comment by jindongjie** [comment]
能提供更详细的配置吗？

**Comment by winsphinx** [comment]
经过一段时间试验，路由策略配 asis 和 ipondemand 都没问题，但ifnotmatch就会偶发上述现象。
具体待进一步验证。

**Reply by RayneScofield** [reply]
跟这个没什么关系吧？

**Reply by iyalue** [reply]
在v2rayN配置里把核心改成sing-box,默认是xray

**Comment by iyalue** [comment]
在v2rayN配置里把核心改成sing-box,默认是xray
